### PR TITLE
Fix cloudinary upload error message

### PR DESCRIPTION
**Pull request by sudhirt4**
This is a minor pull request that fixes the propagation of error from cloudinary to strapi error middleware (err at : boom.Boomify).

Before (when entering invalid cloudinary config)
```
  AssertionError [ERR_ASSERTION]: Cannot wrap non-Error object
```

After
```
Error uploading to cloudinary: Unknown API key 4659829216368531
error Error: Upload to cloudinary failed
```

### CODE REVIEW

**Comment by alexandrebodin** [comment]
Hi @sudhirt4, I don't think these errors should be sent back by the API. We should log them and send a upload error instead so we don't leak too many information

**Comment by sudhirt4** [comment]
@alexandrebodin 
Updated the changes. Now limiting logs from from cloudinary to console while throwing a custom error message.

File: packages/strapi-provider-upload-cloudinary/lib/index.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ module.exports = {
             { resource_type: 'auto', public_id: file.hash, ...customConfig },
             (err, image) => {
               if (err) {
-                return reject(err);
+                console.log(`Error uploading to cloudinary: ${err.message}`);
```
Comment (alexandrebodin) [inline packages/strapi-provider-upload-cloudinary/lib/index.js:22]: here you will need to use strapi.log.error()

Comment (sudhirt4) [reply packages/strapi-provider-upload-cloudinary/lib/index.js]: Updated

**Pull request merged**